### DISCUSSION TITLE
MutScore & MutLand implementation.

### DISCUSSION THREAD

**Discussion by AlessandroLas** [broadinstitute/seqr#3035] [Ideas]
Hi Hana,

I'd like to ask you if there is the possibilty to add MutScore/MutLand to seqr.
The tool could help in prioritizing and classifying VUSs. 

Please find below the link to the publication, to the github page, and to the web app.

https://www.sciencedirect.com/science/article/pii/S0002929722000064

https://github.com/mquinodo/mutscore

https://mutscore-wgt7hvakhq-ew.a.run.app/

Alessandro.

**Comment by hanars** [comment]
seq is an open source system and we welcome code contributions. If your group has the bandwidth to add this functionality to the code base we will happily integrate these changes and make them available to the whole seqr community. However, at this time, we have no plans to add these scores ourselves
